src/fontra/client/core/actions.js

```diff
@@ -61,55 +73,94 @@ export function getActionInfo(actionIdentifier) {
   return actionInfoController.model[actionIdentifier];
 }
 
-export function getActionTitle(actionIdentifier) {
+export function getActionTitle(actionIdentifier, args = "") {
   const actionInfo = getActionInfo(actionIdentifier);
-  return translate(actionInfo?.titleKey || actionIdentifier);
+  return translate(actionInfo?.titleKey || actionIdentifier, args);
 }
 
+// reference: https://www.toptal.com/developers/keycode/table
 export const shortCutKeyMap = {
   ArrowUp: "↑",
   ArrowDown: "↓",
+  ArrowLeft: "←",
+  ArrowRight: "→",
+  Tab: "⇥",
   Delete: "⌫",
+  Backspace: "⌫",
+  NumpadMultiply: "×",
+  NumpadDivide: "÷",
+  NumpadAdd: "+",
+  NumpadSubtract: "-",
+  NumpadEnter: "↵",
+  NumpadDecimal: ".",
+  NumpadEqual: "=",
+  Enter: "↵",
+  Space: "␣",
+  Escape: "Esc",
+  Home: "⌂",
+  End: "End",
+  NumLock: "⌧",
+  PageUp: "⇞",
+  PageDown: "⇟",
+  CapsLock: "⇪",
 };
 
-export function getShortCutRepresentationFromActionIdentifier(actionIdentifier) {
-  if (!actionIdentifier) {
-    return "";
-  }
+// add A-Z keys
+for (const key of new Array(26).fill(1).map((_, i) => String.fromCharCode(65 + i))) {
+  shortCutKeyMap[`Key${key}`] = key;
+}
+// add 0-9 keys
+for (let i = 0; i <= 9; i++) {
+  shortCutKeyMap[`Digit${i}`] = `${i}`;
+  shortCutKeyMap[`Numpad${i}`] = `${i}`;
```

Comment:
I think we should do something like this:
```suggestion
  shortCutKeyMap[`Numpad${i}`] = `Numpad-${i}`;
```
The mechanism distinquishes between "regular" digits and "numpad" digits, and I think we should show that difference.

I would also like to discuss the other Numpad codes/char mapping, such as NumpadMultiply.